docs/multicluster/antctl.md

```diff
@@ -36,57 +36,96 @@ To see the usage examples of these commands, you may also run `antctl mc get [su
+also create a Secret to store the token, as well as a ServiceAccount and a RoleBinding. The `--output-file`
+option saves the member token Secret manifest to a file.
 
 ```bash
-anctcl mc create accesstoken [NAME] [-n NAMESPACE] [--serviceaccount SERVICE_ACCOUNT] [--role-binding ROLE_BINDING]
-antctl mc create clusterclaims [-n NAMESPACE] [--clusterset-id CLUSTERSET_ID] [--cluster-id CLUSTER_ID]
-antctl mc create clusterset [NAME] [-n NAMESPACE] [--leader-server LEADER_SERVER] [--service-account SERVICE_ACCOUNT] [--secret SECRET] [--leader-cluster LEADER_CLUSTER_ID]
+anctcl mc create membertoken [NAME] [-n NAMESPACE] [--output-file OUTPUT_FILE]
 ```
 
 To see the usage examples of these commands, you may also run `antctl mc create [subcommand] --help`.
 
-## antctl mc add
+## antctl mc deploy
+
+`antctl mc deploy` command deploys Antrea Multi-cluster Controller to a leader or member cluster.
 
-`antctl mc add` command can add a new member cluster to a ClusterSet.
++ `antctl mc deploy leadercluster` command deploys Antrea Multi-cluster Controller to a leader cluster and imports
+  all the Antrea Multi-cluster CRDs.
++ `antctl mc deploy membercluster` command deploys Antrea Multi-cluster Controller to a member cluster and imports
+  all the Antrea Multi-cluster CRDs.
 
 ```bash
-antctl mc add membercluster [CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET] [--service-account SERVICE_ACCOUNT]
+antctl mc deploy leadercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
+antctl mc deploy membercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
 ```
 
-To see the usage examples of these commands, you may also run `antctl mc add [subcommand] --help`.
-
-## antctl mc delete
+To see the usage examples of these commands, you may also run `antctl mc deploy [subcommand] --help`.
 
-`antctl mc delete` command can delete resources in an Antrea Multi-cluster ClusterSet.
+## antctl mc init
 
-+ `antctl mc delete clusterclaims` command can delete the two ClusterClaims in a specified Namespace. One for the leader or member cluster, and another for the ClusterSet.
-+ `antctl mc delete clusterset` command can delete a ClusterSet in a leader or member cluster.
-+ `antctl mc delete member-cluster` command can delete a member cluster in a specified Antrea Multi-cluster ClusterSet.
+`antctl mc init` command initializes an Antrea Multi-cluster ClusterSet in a leader cluster. It will create a
+ClusterSet and ClusterClaims for the leader cluster. If the `--output-file` option is specified, the config arguments
+for member clusters to join the ClusterSet will be saved to the specified file.
 
 ```bash
-antctl mc delete clusterclaims [-n NAMESPACE]
-antctl mc delete clusterset [NAME] [-n NAMESPACE]
-antctl mc delete membercluster [MEMBER_CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET]
+antctl mc init [-n NAMESPACE] [--clusterset CLUSTERSET_ID] [--clusterid CLUSTERID] [--create-token] [--output-file OUTPUT_FILE]
```

Comment:
Should we use:

antctl mc init -n NAMESPACE --clusterset CLUSTERSET_ID --clusterid CLUSTERID [--create-token] [--output-file OUTPUT_FILE]

Reply:
Done.